samples/boards/st/mco/boards/nucleo_wba55cg.overlay

```diff
@@ -13,12 +13,14 @@
 	chosen {
 		zephyr,bt-c2h-uart = &lpuart1;
 		zephyr,uart-pipe = &lpuart1;
```

Comment:
Not related but these 2 lines could actually be removed

Reply:
removed